docs/install/installEverest.md

```diff
@@ -0,0 +1,105 @@
+# Install Everest
+
+## Setup
+
+Percona Everest has two primary components:
+
+- [Percona CLI](https://github.com/percona/percona-everest-cli), which connects Percona Everest to your Kubernetes cluster.
+- Everest application (UI for simplicity), which you can spin up using the instructions below.
+
+### Supported operators
+
+- Percona Operator for MySQL Based on Percona XtraDB Cluster (PXC) 1.12
+- Percona Operator for MongoDB (PSMDB) 1.14
+- Percona Operator for PostgreSQL (PG) 2.2
+
+## Before you start
+
+Percona Everest assists with installing all the necessary operators and required packages, but does not currently help with spinning up a Kubernetes cluster.
+Therefore, before installing Everest, ensure that you have a publicly accessible Kubernetes cluster.
+
+We recommend [setting one up on the Amazon Elastic Kubernetes Service (EKS)](../install/eks.md) or on [Google Kubernetes Engine (GKE)](../install/GKE-k8s-cluster), as it may be difficult to currently spin one up on local Kubernetes installations (minikube, kind, k3d, or similar products).
+
+## Prerequisites
+
+Before getting started with Percona Everest, we recommend that you:
+
+1. Install [Docker](https://docs.docker.com/engine/install/) and its compose plugin (included by default in the aforementioned docker engine installation instructions).
+2. Install [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) for EKS or the [gcloud CLI](https://cloud.google.com/sdk/docs/install) for GKE.
+3. Download the kubeconfig file from the Kubernetes cluster that you want to use with Everest. By default, Everest uses the file available under *~/.kube/config.*
```

Comment:
I am not sure about `Download the kubeconfig` because if you create k8s cluster from your local machine, this file will be created automatically.  It is really tricky point because depending on your k8s cluster (EKS/GKE) you need to have additional tool/dependency for it or some additional commands. I suggest to add something like 'You need to have access to k8s cluster' and adding some command on how the user can verify it e.g.
```
❯ kubectl get nodes
NAME                                       STATUS   ROLES    AGE   VERSION
gke-<name>-default-pool-75d48bfc-bx8g   Ready    <none>   11h   v1.26.7-gke.500
gke-<name>-default-pool-75d48bfc-c2df   Ready    <none>   11h   v1.26.7-gke.500
gke-<name>-default-pool-75d48bfc-zl7k   Ready    <none>   11h   v1.26.7-gke.500
```